elements/pfe-band/demo/index.html

```diff
@@ -193,29 +203,34 @@ <h2>Darkest band with header tag</h2>
       </header>
       <section class="body1 generic1 band">
       <p>Consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>
-      <pfe-cta pfe-priority="secondary" slot="pfe-band--footer">
+<footer slot="pfe-band--footer">
+      <pfe-cta pfe-priority="primary" >
         <a href="#">Learn more</a>
       </pfe-cta>
+</footer>
     </section>
       <pfe-card pfe-color="darker" slot="pfe-band--aside">
-        <h3 slot="pfe-card--header">Aside: right full top</h3>
+        <h3 slot="pfe-card--header">Aside: right full top, darker card</h3>
```

Comment:
#pebble 
```suggestion
        <h3 slot="pfe-card--header">Aside: right full top, Darker card</h3>
```

To match the other card capitalizations 🤷 #lowpriority

Reply:
github wouldn't let me apply the suggestion here but i fixed it in the file